nyx_client/nyx_client/client.py

```diff
@@ -647,6 +651,9 @@ def create_data(
         elif access_control == "none":
             data["accessControl"] = [ALLOW_NONE]
 
+        if connection_id:
```

Comment:
(pedantic) - unit tests for this & get_connections? (Or are we not doing those here? :thinking: )